src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/CalculateEarnings/CalculateEarningsCommandValidator.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Commands.Types.ApprenticeshipIncentive;
+
+namespace SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.CalculateEarnings
+{
+    public class CalculateEarningsCommandValidator : IValidator<CalculateEarningsCommand>
```

Comment:
Are these validators actually necessary?  We're protecting from ourselves as these will never be based on user input.

Reply:
Just following the pattern.  The command handler doesn't know how or who entered the command data so is just protecting it's invariants.

Reply:
Just seems like unnecessary code given these are handling domain events from the same solution.

Reply:
Updated to not be required for DomainCommands.